src/app/replayer/replayer.ml

```diff
@@ -1601,6 +1660,14 @@ let () =
            Param.flag "--output-file"
              ~doc:"file File containing the resulting ledger"
              Param.(optional string)
+         and migration_mode =
+           Param.flag "--migration-mode"
```

Comment:
Can we rename this to `berkeley-migration-mode`? it is very specific to berkeley upgrade